### DISCUSSION TITLE
feature request: change backgroud color or theme by account

### DISCUSSION THREAD

**Discussion by m-tmatma** [community/community#142440] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, バージョン 1.183.0 (144611670)
Device: iPhone 15 Pro (iOS 18.0.1)
</details>

GitHub mobile app can switch accouts, but the appearance is same after changing accounts.
So it's not clear what account we are using.
If there is feature to choose background color or theme in account basis, we can know it immediately.

Regards

**Comment by taki-on** [comment]
@m-tmatma Thank you for an interesting suggestion! Our designers would interested in your idea 😄 In the meantime, you can check the active account through the avatar you can find in the tab bar. I hope it is helpful. Thank you!

**Comment by m-tmatma** [comment]
@taki-on Thank you for replying.
It worked, I’m looking forward to supporting my suggestion.
Regards.

**Comment by sottototto** [comment]
> ℹ️ Additional Debug Info
> GitHub mobile app can switch accouts, but the appearance is same after changing accounts. So it's not clear what account we are using. If there is feature to choose background color or theme in account basis, we can know it immediately.
> 
> Regards
